app/models/user.rb

```diff
@@ -1,4 +1,10 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As requested by the previous reviewer, kindly enable `confirmable` in the user model. The user should be prompted to confirm their account before being signed in upon signing up.

```
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```